backend/src/main/resources/graphql/main.graphqls

```diff
@@ -30,10 +30,6 @@ type Facility {
   phone: String
   email: String
   deviceTypes: [DeviceType!]!
-  defaultDeviceSpecimen: ID
-  deviceSpecimenTypes: [DeviceSpecimenType]
-    @deprecated(reason: "kept for compatibility")
-  defaultDeviceType: DeviceType @deprecated(reason: "kept for compatibility")
```

Comment:
non-blocking question: any idea why these were kept for compatibility and now they're not needed? had they just not been removed from certain queries?

Reply:
my guess it was left here during a change to maintain backward compatibility with an earlier commit, but it was neglected to be cleaned up afterwards, we're not generally great at that, I keep cleaning up old graphql endpoints.

Reply:
thanks for doing the cleanup 👍